src/Activity/AdverbsData.js

```diff
@@ -0,0 +1,79 @@
+import React from "react";
```

Comment:
The indentation in this file is terrible. Please try to install Prettier and Eslint in your text editor and turn on auto formatting. The project uses 2 spaces for tabulation